_config.yml

```diff
@@ -37,15 +37,15 @@ amy_site: "https://amy.software-carpentry.org/workshops"
 carpentries_github: "https://github.com/carpentries"
 carpentries_pages: "https://carpentries.github.io"
 carpentries_site: "https://carpentries.org/"
-dc_site: "http://datacarpentry.org"
+cc_by_human: "https://creativecommons.org/licenses/by/4.0/"
```

Comment:
Can you give some background on these changes? My understanding was that `_config.yml` should match https://github.com/carpentries/styles/blob/gh-pages/bin/boilerplate/_config.yml

Reply:
yes that's correct. However, for `_config.yml`, we don't have a good automated mechanism to do this, and it's possible that the content of the file in this repo diverged from the one in the `styles` repository. Re: the changes proposed here, it looks OK. There is no need to remove `example_repo` and `example_lesson` though.

Reply:
Each of these changes was either a duplicate or linked to a breaking missing variable in the _includes that I brought across from the generic.  
The one exception was changing `dc_site`. I could not find a clear reason for it to be using http:// instead of https:// and testing it out several ways showed the https:// not to be breaking (http:// redirects to https:// anyway).